### ISSUE TITLE
storybook requires is invalid on windows as of 8.4.x

### ISSUE THREAD

**Issue by trajano** [storybookjs/react-native#645]
**Describe the bug**

```
/* do not change this file, it is auto generated by storybook. */

import { start, updateView } from "@storybook/react-native";

import "@storybookaddon-ondevice-notes\register";
import "@storybookaddon-ondevice-controls\register";
import "@storybookaddon-ondevice-actions\register";
import "@storybookaddon-ondevice-backgrounds\register";
```
Note the missing `/` and the incorrect `\`

**Comment by trajano** [comment]
I am thinking it's something to do with https://github.com/storybookjs/react-native/commit/467b40ba6dfec88eb77469c1d5850470fefebe55 as this changed generate.js last week

**Comment by trajano** [comment]
The patch I have corrects the common.js which fixes the generated file but not during runtime

**Comment by dannyhw** [comment]
Ah i know the problem i think, path.join is killing me

**Comment by trajano** [comment]
I couldn't find the part where you do it at runtime but just adding `.posix` after `path` should be sufficient.

**Comment by dannyhw** [comment]
If I put out a prerelease version are you able to test this easily?

**Comment by dannyhw** [comment]
@trajano can you test `8.4.2-alpha.0`

**Comment by dannyhw** [comment]
should be fixed now in 8.4.2
